### PR TITLE
Header

### PR DESCRIPTION
**Pull request by Josetf2396**
image

### CODE REVIEW

**Review by raynesmike — APPROVED** [review]
Everything seems good, We just need to update couple things once theme is added

File: mobile/src/components/header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  statusBarHidden: boolean;
+  rightComponentStyle: any;
+  leftComponentDisable: boolean;
+  rightComponentDisable: boolean;
+  leftComponentOnPress: () => void;
+  rightComponentOnPress: () => void;
+  leftIconName: any;
+  rightIconName: any;
+  leftIconStyle: any;
+  rightIconStyle: any;
```
Comment (carizzuti) [inline mobile/src/components/header/Header.tsx:32]: Consider not using `any` type. With this a string or number can be passed in with no error.

**Review by Nap1400 — APPROVED** [review]

File: mobile/src/components/header/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  View, 
+  Text, 
+  TouchableOpacity } from "react-native";
+import {Icon} from '../icon/Icon';
+import styles, {
+  _container,
+  _innerContainer,
+} from "./Header.style";
+
+interface IProps {
```
Comment (lasharela) [inline mobile/src/components/header/Header.tsx:13]: why we need interface and not type... if you are trying to extend existing type of the header that's not the way to go. we don't really need to extend and even if we need, we have to do this differently. try to google how to extend typescript types and interfaces.

**Review by lasharela — CHANGES_REQUESTED** [review]

**Comment by lasharela** [comment]
Visually it looks more complicated

**Review by carizzuti — APPROVED** [review]

**Pull request merged**